SPECS/sdbus-cpp/sdbus-cpp.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "sdbus-cpp-1.1.0.tar.bz2": "62eef2197d5079206e103573614feee72c6831fe599a1f5880413a64e8b8fca7"
```

Comment:
you should update the signature and the extension (.tar.gz) since you're now using tar.gz file

Reply:
Fixed